### DISCUSSION TITLE
203

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#184] [General]
# 潮流周刊第203期 - 银杏季节

封面图拍摄于周末西湖 Apple 门口的银杏树，这个季节杭州的银杏非常好看，特别是蓝天太阳照在上面的时候，很有美感，就是最近突然有些冷了，不太习惯，有一种夏天一下子转眼就冬天那种感觉。

https://weekly.tw93.fun/posts/203-%E9%93%B6%E6%9D%8F%E5%AD%A3%E8%8A%82

**Comment by Th2esh** [comment]
保持理智

**Comment by ChengWeiDong** [comment]
相信未来，世界会好的

**Comment by Rickyovo** [comment]
MUJI Labo 防风抗水裤，淘宝看了下双十二到手200元，黑色款缺码比较厉害
